clang/include/clang/CIR/Common/ABIArgInfo.h

```diff
@@ -0,0 +1,226 @@
+//==-- ABIArgInfo.h - Abstract info regarding ABI-specific arguments -------==//
+//
```

Comment:
I think the `Common` sub path isn't necessary, `clang/include/clang/CIR/ABIArgInfo.h` should be good enough